src/main/java/internet/shop/services/interfaces/ProductService.java

```diff
@@ -0,0 +1,19 @@
+package internet.shop.services.interfaces;
+
+import internet.shop.models.Product;
+import java.util.List;
+import java.util.Optional;
+
+public interface ProductService {
+    Product create(Product product);
+
+    Optional<Product> getById(Long productId);
```

Comment:
you should have the following methods here:

- Product create(Product product);
- Product get(Long id);
- List<Product> getAll();
- Product update(Product product);
- boolean delete(Long id);